ls1intum/Artemis
# [Enhancements] - CSS fixes and statistics reordering

### Checklist

### Motivation and Context


Some minor css bugs still occur in the current version. Also, the exercises in the course statistics are a set and therefore not explicitly sorted

### Description

- Moved the "register" and the "create new course" buttons to the places they used to be before 5.0
- Sorted the exercises in the course statistics after releaseDate

### Steps for Testing


1. Log in to Artemis
2. Navigate into the student course overview
3. Check that the button to register to a new course is like in the screenshot, on the top right
4. Navigate into the course management overview
5. Check that the button to create a new course is like in the screenshot, on the top right
6. Go to the course statistics and check that the exercises shown in the average scores are sorted after release dates

### Test Coverage




### Screenshots


Course overview before:
![Screenshot 2021-06-29 at 21 15 58](https://user-images.githubusercontent.com/44401560/123856678-aa586500-d921-11eb-8c14-9f4f183bdd78.png)

Course overview after:
![Screenshot 2021-06-29 at 21 22 14](https://user-images.githubusercontent.com/44401560/123856715-b512fa00-d921-11eb-8a24-6ba16a66d8ed.png)

Course management overview before:
![Screenshot 2021-06-29 at 21 24 03](https://user-images.githubusercontent.com/44401560/123856744-bc3a0800-d921-11eb-85a2-683f7c164fe7.png)

Course management overview after:
![Screenshot 2021-06-29 at 21 27 06](https://user-images.githubusercontent.com/44401560/123856757-c22fe900-d921-11eb-9a1f-9a340e6795a1.png)
